react/css/src/digitv2/pages/employee/inboxsearchcomposer.scss

```
@@ -320,74 +320,74 @@
         }
 
         .digit-inbox-search-links-header-text {
             @include typography-heading-m;
-            font-family: theme("digitv2.fontFamily.sans");
-            font-style: theme("digitv2.fontStyle.normal");
-            font-weight: theme("digitv2.fontWeight.bold");
-            line-height: theme("digitv2.lineHeight.lineheight1");
+            font-family: var(--digitv2-fontFamily-sans);
+            font-style: var(--digitv2-fontStyle-normal);
+            font-weight: var(--digitv2-fontWeight-bold);
+            line-height: var(--digitv2-lineHeight-lineheight1);
```

Stylelint (17.3.0)

[error] Unexpected duplicate "line-height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)